src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -20,6 +20,8 @@ public sealed partial class JsonSerializerOptions
 
         private readonly ConcurrentDictionary<Type, JsonClassInfo> _classes = new ConcurrentDictionary<Type, JsonClassInfo>();
 
+        // For any new option added, adding it to the options copied in the copy constructor below must be considered.
```

Comment:
For a few places ASP.NET Core had to write a copy ctor, we have found it useful to add a reflection based test: https://github.com/dotnet/aspnetcore/blob/master/src/Mvc/Mvc.Core/test/ApplicationModels/ActionModelTest.cs#L60.

Reply:
Nice, thanks!